src/main/handlers/leaves.py

```diff
@@ -0,0 +1,70 @@
+import asyncio
+import random
+
+from datetime import datetime, timedelta
+
+import discord
+
+
+ONGOING = 'ongoing'
+MESSAGE_COUNT = 'message_count'
+NUM_WAIT_MESSAGES = 'number_of_messages_to_wait_for'
+LEAVES_SENT = 'leaves_sent'
+TIMES_REPEAT = 'times_to_repeat'
+LAST_LEAVES = 'last_leaves'
+
+class Leaves:
+
+    def __init__(self, client, config):
+        self.client = client
+        self.config = config
+
+        self.leaves_emoji = '🍃'
+        self.leaves_alias = ':leaves:'      
+        self.leaves_destroyer = ':fire:'
+        self.leaves_sources = ['🍃', '🌳']  
+        self.cooldown = 30
+
+        self.server_vars = {}
+
+    async def execute(self, message):
+        if not message.guild.id in self.server_vars:
+            self.server_vars[message.guild.id] = {}
```

Comment:
why do these values need to be stored in a dict? it seems simpler to just put them in their own variables

Reply:
It's set up to handle several Discord servers so the different values need to be stored somewhere else.